app/src/interfaces/map/map.vue

```diff
@@ -105,7 +105,7 @@ export default defineComponent({
 		},
 		geometryType: {
 			type: String as PropType<GeometryType>,
-			default: undefined,
+			default: 'GeometryCollection',
```

Comment:
This wont work with the rest of the file. (See `getDrawOptions`, `loadValueFromProps`, `isTypeCompatible` and most likely elsewhere).